src/server/routes/actions.spec.ts

```diff
@@ -0,0 +1,279 @@
+// we need the mock methods to be before the imports for jest to run
```

Comment:
Would it be a good idea to move this exception to `.eslintrc.cjs` (or to `opstooling-js-style` eventually)? (If the file matches `*.spec.ts`, then we disable `import/first`).

Reply:
I would say no, let's do it in a per basis case, as in most cases we don't really mock imports, or we are able to mock it in a more effective way.

The only reason for which I had to do this is because the imports initialize code instead of having an initializer, in any other case I would be able to simply mock later.